Testscripts/Linux/KDUMP-Config.sh

```diff
@@ -0,0 +1,401 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+kdump_conf=/etc/kdump.conf
+dump_path=/var/crash
+sys_kexec_crash=/sys/kernel/kexec_crash_loaded
+kdump_sysconfig=/etc/sysconfig/kdump
+boot_filepath=""
+
+#
+# Source utils.sh to get more utils
+# Get $DISTRO, LogMsg directly from utils.sh
+#
+. utils.sh || {
+    echo "unable to source utils.sh!"
+    exit 0
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+#######################################################################
+#
+# InstallKexec()
+#
+#######################################################################
+InstallKexec(){
+    GetDistro
+    case $DISTRO in
+        centos* | redhat* | fedora*)
+            yum install -y kexec-tools kdump-tools makedumpfile
+            if [ $? -ne 0 ]; then
+                UpdateSummary "Warning: Kexec-tools failed to install."
+            fi
+        ;;
+        ubuntu* | debian*)
+            export DEBIAN_FRONTEND=noninteractive
+            apt-get update; apt-get -y install kexec-tools kdump-tools makedumpfile
+            if [ $? -ne 0 ]; then
+                UpdateSummary "Warning: Kexec-tools failed to install."
+            fi
+        ;;
+        suse*)
+            zypper refresh; zypper --non-interactive install kexec-tools kdump makedumpfile
+            if [ $? -ne 0 ]; then
+                UpdateSummary "Warning: Kexec-tools failed to install."
+            fi
+        ;;
+        *)
+            msg=
+            LogErr "Warning: Distro '${distro}' not supported. Kexec-tools failed to install."
```

Comment:
Replace with LogWarn

Reply:
LogWarn is not available for bash scripts, only for ps1 afaik